ansible/roles/java-base/tasks/main.yml

```diff
@@ -34,11 +34,11 @@
 # if this fails you want to check in vars/main.yml and add package name
 # as appropriate -- try to use generic os family if available.
 - name: install java
-  when: not os|startswith("zos")
+  when: not os|startswith("zos") and "ppc64" not in ansible_nodename
```

Comment:
This just seems to not install the packages on the ppc64 machines I don't think that is what we want?